spec/wings/orm_converter_spec.rb

```diff
@@ -48,6 +48,7 @@
 
     context 'when given a non-ActiveFedora class' do
       it 'raises an exception' do
+        # TODO: What error are we expecting to raise
```

Comment:
would it be worth just putting in whatever error is actually raised now, as a first step?

Reply:
It would be…except I didn't have band-width to fix it. I was more interested in getting this flappy test, which has raised its head quite a bit today, resolved.

Reply:
With it resolved, I can look at the exceptions (though I'm trying to tackle why CircleCI failed in a different spot and required a restart)

Reply:
I went ahead and pushed those changes.